src/index.js

```diff
@@ -6,17 +6,16 @@ import { browserHistory } from 'react-router';
 import Routes from './routes';
 import thunk from 'redux-thunk';
 import createLogger from 'redux-logger';
+
 import rootReducer from './reducers/rootReducer';
```

Comment:
Agreed, applyMiddleware was put in place by default in the original repo we're all using!